sdk/core/azure-core/src/credentials/credentials.cpp

```diff
@@ -49,8 +49,9 @@ AccessToken Azure::Core::Credentials::ClientSecretCredential::GetToken(
   static std::string const errorMsgPrefix("ClientSecretCredential::GetToken: ");
   try
   {
-    Http::Url url;
-    url.SetPath(m_authority + UrlEncode(m_tenantId) + "/oauth2/v2.0/token");
+    Http::Url url(m_authority);
```

Comment:
Tests?

Reply:
entire feature (Token Credential policy) is only covered with e2e tests from Storage for now.